BSP/Inc/BSP_I2C.h

```diff
@@ -14,22 +15,22 @@ void BSP_I2C_Init(void);
 
 /**
  * @brief   Transmits data onto the I2C bus.
- * @param   deviceAddr : the device/IC that the data is intended for.
+ * @param   devAddr : the device address to write to
```

Comment:
Why did we change the parameter name from `deviceAddr` to `devAddr` the first one seems better to me.